test/e2e/config/fixture.yaml

```diff
@@ -31,6 +31,7 @@ intervals:
   inspection/wait-inspecting: ["5s", "10ms"]
   inspection/wait-available: ["5s", "1ms"]
   external-inspection/wait-available: ["5s", "1ms"]
+  re-inspection/wait-available: ["5m", "1s"]
```

Comment:
I added a new one because the others were very small and re-inspection was taking some time. Should I change `default/wait-available:` in both `fixture` and `ironic` to `["10m", "10s"]` and use that instead?

Reply:
Fixture is different it should not take much time. Let me check!

Reply:
available after re-inspection almost takes 3 minutes I think:
```
  STEP: waiting for the BMH to become available after re-inspection @ 10/31/23 08:40:01.681
  STEP: checking that the hardware details are corrected after re-inspection @ 10/31/23 08:42:54.127
```

Reply:
For fixture it takes less than a second most of the time.
```
  STEP: adding InspectAnnotation to re-inspect @ 10/31/23 10:54:30.718
  STEP: waiting for the BMH to be in inspecting state after inspection annotaion @ 10/31/23 10:54:30.739
  STEP: waiting for the BMH to become available after re-inspection @ 10/31/23 10:54:30.756
  STEP: checking that the hardware details are corrected after re-inspection @ 10/31/23 10:54:30.961
```

Fixture is not what runs of you use the `ci-e2e.sh` script. If you want to run it you need to check the bottom of the readme here: https://github.com/metal3-io/baremetal-operator/blob/main/test/e2e/README.md

Reply:
One more thing! If you run into any issues running the tests with the fixture provider on Mac, please bring them up! I want this to be easy for developers to run :slightly_smiling_face:

Reply:
I was running tests in VM but I will try on my mac too.